src/cookbook/design/themes.md

```diff
@@ -53,89 +86,113 @@ MaterialApp(
+## Apply a theme
 
-  <iframe class="full-width" src="{{site.youtube-site}}/embed/oTvQDJOBXmM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-{{site.alert.end}}
+To apply your new theme, add the `Theme.of(context)` method within a widget's styling properties. These include, but are not limited to, `style` and `color`.
+
+The `Theme.of(context)` method looks up the widget tree and retrieves
+the nearest `Theme` in the tree.
+If you have a standalone `Theme` as defined in the previous example,
+that's applied.
+If not, Flutter applies the app's theme.
+
+In fact, the `Container` uses this technique to find its `color``.
+
+<?code-excerpt "lib/main.dart (Container)" replace="/^child: //g"?>
+```dart
+Container(
+  padding: const EdgeInsets.symmetric(
+    horizontal: 12,
+    vertical: 12,
+  ),
+  color: Theme.of(context).colorScheme.primary,
+  child: Text(
+    'Text with a background color',
+    // ···
+    style: Theme.of(context).textTheme.bodyMedium!.copyWith(
+          color: Theme.of(context).colorScheme.onPrimary,
+        ),
+  ),
+),
+```
+
+## Override a theme
+
+To override the overall theme in part of an app,
+wrap that section of the app in a `Theme` widget.
 
-### Creating unique `ThemeData`
+You can override a theme in two ways:
 
-If you don't want to inherit any application colors or font styles,
-create a `ThemeData()` instance and pass that to the `Theme` widget.
+1. Create a unique `ThemeData` instance.
+2. Extend the parent theme.
 
-<?code-excerpt "lib/theme.dart (Theme)"?>
+### Set a unique `ThemeData` instance
+
+If you want a component of your app to ignore the overall theme,
+create a `ThemeData()` instance.
+Pass that instance to the `Theme` widget.
+
+<?code-excerpt "lib/main.dart (Theme)"?>
 ```dart
 Theme(
-  // Create a unique theme with `ThemeData`
+  // Create a unique theme with `ThemeData`.
   data: ThemeData(
-    splashColor: Colors.yellow,
-  ),
+      colorScheme: ColorScheme.fromSeed(
+    seedColor: Colors.pink,
+  )),
   child: FloatingActionButton(
     onPressed: () {},
     child: const Icon(Icons.add),
   ),
 );
 ```
 
-### Extending the parent theme
+### Extend the parent theme
 
-Rather than overriding everything, it often makes sense to extend the parent
-theme. You can handle this by using the [`copyWith()`][] method.
+Instead of overriding everything, consider extending the parent theme.
+To extend a theme, use the [`copyWith()`][] method.
 
-<?code-excerpt "lib/theme.dart (ThemeCopyWith)"?>
+<?code-excerpt "lib/main.dart (ThemeCopyWith)"?>
 ```dart
 Theme(
-  // Find and extend the parent theme using `copyWith`. See the next
-  // section for more info on `Theme.of`.
-  data: Theme.of(context).copyWith(splashColor: Colors.yellow),
+  // Find and extend the parent theme using `copyWith`.
+  // To learn more, check out the next section on `Theme.of`.
```

Comment:
Since you moved the `Theme.of` section before this, either say "preceding section" or just drop this line.